app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class QuestionsController < ApplicationController
+  before_action :set_question!
```

Comment:
empty line

Reply:
we use bang at the end of the method if this method does destructive changes to DB, this is not that situation